Moved scenario two sentence to an important note in decommission-on-premises-exchange.md (#2878)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/decommission-on-premises-exchange.md

### Scenario two

 **Issue**: My organization has been running in a hybrid configuration for about a year now and has finally moved my last mailbox to the cloud. I plan to keep Active Directory Federation Services (AD FS) for user authentication of my Exchange Online mailboxes.
 
> [!IMPORTANT]

> > This scenario applies to any customer that is planning on keeping directory synchronization.

Reviewer: ```suggestion
> This scenario applies to any customer who is planning to keep directory synchronization.
```
